pytorch_lightning/callbacks/stochastic_weight_avg.py

```diff
@@ -16,18 +16,25 @@
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 """
 from copy import deepcopy
-from typing import Callable, List, Optional, Union
+from typing import Any, Callable, Dict, List, Optional, Union
 
 import torch
 from torch import nn
 from torch.optim.swa_utils import SWALR
 
 import pytorch_lightning as pl
 from pytorch_lightning.callbacks.base import Callback
+from pytorch_lightning.plugins.training_type.deepspeed import DeepSpeedPlugin
+from pytorch_lightning.plugins.training_type.fully_sharded import DDPFullyShardedPlugin
 from pytorch_lightning.trainer.optimizers import _get_default_scheduler_config
 from pytorch_lightning.utilities import rank_zero_info, rank_zero_warn
 from pytorch_lightning.utilities.exceptions import MisconfigurationException
 
+try:
+    from pytorch_lightning.plugins.training_type.fully_sharded import FullyShardedDataParallel
+except ImportError:
+    FullyShardedDataParallel = None
```

Comment:
You are importing the wrong thing. From the code below it seems you want to import the plugin. This can be done without checking for import error.

```suggestion
from pytorch_lightning.plugins.training_type import DDPFullyShardedPlugin
```

Reply:
Whoops, thanks yes I'm already importing DDPFullyShardedPlugin above, not sure why I thought I needed this.